projenrc/github-workflows.ts

```diff
@@ -267,11 +267,11 @@ export function buildOrtToolkitWorkflow(project: AwsCdkConstructLibrary) {
     const workflow = project.github.addWorkflow('ort-toolkit');
     if (workflow) {
       workflow.on({
-        push: {
-          branches: [
-            'main',
-          ],
-        },
+        // push: {
```

Comment:
Delete this workflow?

Reply:
Eventually, this should run automatically on only the `main` branch. Watching these:

* https://github.com/oss-review-toolkit/ort/issues/6465
* https://github.com/oss-review-toolkit/ort/pull/7494

The results are inconsistent:
* https://github.com/awslabs/generative-ai-cdk-constructs/actions/runs/7040789381
* https://github.com/awslabs/generative-ai-cdk-constructs/actions/runs/7023997394